src/telemetry/logs.ts

```diff
@@ -60,16 +60,32 @@ export class GlobalLogger {
     this.addContextMetadata = config?.addContextMetadata || false;
   }
 
-  info(message: string, metadata?: ContextualMetadata): void {
-    this.logger.info(message, metadata);
+  // We use this form of winston logging methods: `(message: string, ...meta: any[])`. See node_modules/winston/index.d.ts
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  info(logEntry: any, metadata?: ContextualMetadata): void {
+    if (typeof logEntry === "string") {
+      this.logger.info(logEntry, metadata);
+    } else {
```

Comment:
Is this what Winston does by default when it receives non-string input?  What were we doing before?

Reply:
Here's the set of inputs Winston accepts for its log methods:

```
 interface LeveledLogMethod {
      (message: string, callback: LogCallback): Logger;
      (message: string, meta: any, callback: LogCallback): Logger;
      (message: string, ...meta: any[]): Logger;
      (message: any): Logger;
      (infoObject: object): Logger;
    }
```

We have always been using either `(message: string, ...meta: any[]): Logger;` or `(message: any): Logger;`